src/OrchardCore.Modules/OrchardCore.Sitemaps/Views/CustomPathSitemapSource.Edit.cshtml

```diff
@@ -0,0 +1,48 @@
+@model CustomPathSitemapSourceViewModel
```

Comment:
This require @using OrchardCore.Sitemaps.Builders namespace or add the using to the _ViewImports.cshtml file of that module. Else when building it won't find the reference to these ViewModels.

Reply:
@Skrypt I believe this @using is not necessary. This class is 'CustomPathSitemapSourceViewModel' and it is under **ViewModel** folder, not under **Builders** folder.

Reply:
Well maybe I did not put my comment on the right line. But definitely you need to add the `@using OrchardCore.Sitemaps.Builders` in the _ViewImport.cshtml file else it won't publish your website. So this breaks "publishing" of Orchard Core.

Reply:
Actually I think you are right. This is maybe a bad build/publish on my side.